assets/src/application/ui/input/Button/IconButton/types.ts

```diff
@@ -0,0 +1,11 @@
+import { withLabelProps, withTooltipProps } from '../../../display';
+import { IconButtonProps as IconButtonAdapterProps } from '@infraUI/inputs';
+
+export interface IconButtonProps
+	extends Omit<IconButtonAdapterProps, 'aria-label'>,
+		Partial<withLabelProps>,
+		Partial<withTooltipProps> {
+	'aria-label'?: string;
+	borderless?: boolean;
+	color?: 'white' | 'black';
```

Comment:
hmmmm... good for now but will change later... black is usually too strong of a contrast, but not going to worry about it now

Reply:
yes, that was just to have smth as an example, will adjust in the future as per our needs